egs2/TEMPLATE/asr1/scripts/utils/show_asr_result.sh

```diff
@@ -44,17 +44,17 @@ cat << EOF
 EOF
 
 while IFS= read -r expdir; do
-    if ls "${expdir}"/*/*/score_*/result.txt &> /dev/null; then
+    if ls "${expdir}"/*/score_*/result.txt &> /dev/null; then
         echo "## $(basename ${expdir})"
         for type in wer cer ter; do
-            if ls "${expdir}"/*/*/score_${type}/result.txt &> /dev/null; then
+            if ls "${expdir}"/*/score_${type}/result.txt &> /dev/null; then
```

Comment:
The previous `/*/*/` will failed when use `exp` parameter to specify a directory. I removed a `*`, now it look ok.

Reply:
I did modify asr.sh long time ago to work with */* for readability. Please keep as it is and should fix enh_asr.sh rather.

https://github.com/espnet/espnet/blob/e42b1ef56b433266e2ed51e87d7060982ae7d27a/egs2/TEMPLATE/asr1/asr.sh#L1185

Reply:
Before this modification, I have also referred to `asr.sh`, and run `./scripts/utils/show_asr_result.sh exp/some_exp_dir`, but found it not work in asr task either.  After remove a `*`, then both tasks works fine. I'm not sure if it is something else wrong in my environment. Could you please try it in the asr task?

Reply:
It works fine for me. If the current code doesn't work really, the current ci should also fail, so something is wrong in your side.

Please check asr.sh again. 

`${asr_exp}/${inference_tag}/${dset}/score_${_type}` is match with `"${expdir}"/*/*/score_${type}`.

Reply:
The expected directory structure is as following:

```
- exp
   -  asr_train*
       -  inference*
           -  eval_set1
               -  score_cer
               -  score_wer
           -  eval_set2
               -  score_cer
               -  score_wer
           - ...

```

Reply:
Thank you @kamo-naoyuki ! I find the problem. The ${inference_tag} is missing in my code base, thus the directory structure is one-level shallower. I'll fix this, Thank you!